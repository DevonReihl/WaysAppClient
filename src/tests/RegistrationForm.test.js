import React from 'react'
import RegistrationForm from '../components/RegistrationForm/RegistrationForm'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'

describe(`RegistrationForm component`, () => {
    it('Renders without crashing', () => {
        shallow(<RegistrationForm />)
    })
    it('Matches snapshot', () => {
        const wrapper =  shallow(
        <RegistrationForm  />
        )
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})