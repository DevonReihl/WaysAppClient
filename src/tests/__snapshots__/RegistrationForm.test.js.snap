// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationForm component Matches snapshot 1`] = `
<div
  className="form-wrapper"
>
  <form
    action="#"
    className="LoginForm"
    onSubmit={[Function]}
  >
    <div
      role="alert"
    />
    <div>
      <Label
        htmlFor="registration-name-input"
      >
        Enter your name
        <Required />
      </Label>
      <br />
      <ForwardRef
        id="registration-name-input"
        name="name"
        required={true}
      />
    </div>
    <div>
      <Label
        htmlFor="registration-username-input"
      >
        Choose a username
        <Required />
      </Label>
      <br />
      <ForwardRef
        id="registration-username-input"
        name="username"
        required={true}
      />
    </div>
    <div>
      <Label
        htmlFor="registration-password-input"
      >
        Choose a password
        <Required />
      </Label>
      <br />
      <ForwardRef
        id="registration-password-input"
        name="password"
        required={true}
        type="password"
      />
    </div>
    <div>
      <div
        className="button-wrapper"
      >
        <Link
          replace={false}
          to="/login"
        >
          Already have an account?
        </Link>
        <br />
        <br />
        <ForwardRef
          className="myButton"
          type="submit"
        >
          Sign up
        </ForwardRef>
         
      </div>
    </div>
  </form>
</div>
`;
